vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -396,6 +396,18 @@ components:
           default: false
       description: Virtual machine configuration
 
+    CpuTopology:
```

Comment:
Based on your description a per b, b per c, c per d, that means you expect always symmetrical topology. It sounds fine to me, but I'm wondering if a user might expect a machine with one socket being different from the other, which in this case he won't be able to model.